core/src/main/scala/org/allenai/common/cache/QueryCache.scala

```diff
@@ -0,0 +1,66 @@
+package org.allenai.common.cache
+
+import org.allenai.common.Logging
+
+import redis.clients.jedis.{ Jedis, JedisPool }
+
+import spray.json._
+
+/** class holding a Redis cache of query results. This is meant to store any value T where
```

Comment:
You should probably update this to mention that it stores anything that has a JSON serialization.